UKHomeOffice/gro
# Integrate latest confirm controller and emailer

- remove old confirm controller and view
- remove confirm-step-config
- use latest confirm controller
- update email configuration to use SES API
- make sure correct fields rendered to confirm view
- remove annoying console log form model postcode test
- fixed GA TAG ENV VAR reference
